metrics_utility/management/commands/build_report.py

```diff
@@ -208,6 +210,11 @@ def _handle_extra_params(self, ship_target=None):
         )
         return base
 
+    def _next_month(self, date):
+        if date.month == 12:
+            return date.replace(month=1, year=date.year + 1)
+        return date.replace(month=date.month + 1)
```

Comment:
date + relativedelta(months=1) should result in the same?

Reply:
oh, thanks! timedelta didn't have months, missed relativedelta :)